test/general/signature.js

```diff
@@ -650,29 +650,29 @@ hUhMKMuiM3pRwdIyDOItkUWQmjEEw7/XmhgInkXsCw==
 -----END PGP SIGNATURE-----
 `;
 
-  it("Retrieve the issuer Key ID of a signature", async function () {
-    const { privateKeyArmored, publicKeyArmored } = await openpgp. generateKey({
-        type: "ecc", // Type of the key, defaults to ECC
-        curve: "curve25519", // ECC curve name, defaults to curve25519
-        userIDs: [{ name: "name", email: "test@email.com"}], // you can pass multiple user IDs
-        passphrase: "password", // protects the private key
-      });
+  it("Retrieve the issuer ID of a signature", async function () {
```

Comment:
They are really called Key IDs :) See https://datatracker.ietf.org/doc/html/rfc4880#section-3.3.
```suggestion
  it("Retrieve the issuer Key ID of a signature", async function () {
```